app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
Why are you sending timezone unaware datetime in the first place, currently we are accepting timezone aware datetime and giving it back. Please have a look at the recent timezone PRs. Changetotimezone is not needed.